tests/testsuite/directory.rs

```diff
@@ -418,3 +418,62 @@ fn directory_in_git_repo_truncate_to_repo_true() -> io::Result<()> {
     assert_eq!(expected, actual);
     Ok(())
 }
+
+#[test]
+#[ignore]
+fn use_logical_and_physical_paths() -> io::Result<()> {
+    /* This test is a bit of a smoke + mirrors trick because all it shows is that
+    the application is reading the PWD envar correctly (if the shell doesn't
+    correctly set PWD, we're still in trouble). */
+    let tmp_dir = Path::new("/tmp/starship/porthole/viewport");
+    let dir = tmp_dir.join("directory");
+    let sym = tmp_dir.join("symlink_to_directory");
+    fs::create_dir_all(&dir)?;
+    std::os::unix::fs::symlink(&dir, &sym).unwrap();
```

Comment:
Could we add conditional compilation to use `std::os::windows::fs::{symlink_file, symlink_dir}` so that we don't have to revisit this test when adding Windows support?

Reply:
Can do! I just have no idea how to test it...

Reply:
@matchai Done, but for now I'm just going to have to hope that this does what I want on Windows.